features/gezag-minderjarige/1.1-ingezetene-BRP.feature

```diff
@@ -0,0 +1,95 @@
+  gezagsmutatie wordt bijvoorbeeld niet bijgewerkt in de RNI.
+
+  Een persoon is "ingezetene" en "ingeschreven in de BRP" wanneer deze staat ingeschreven
+  in een Nederlandse gemeente. Een persoon is "niet ingezetene" wanneer deze staat ingeschreven 
+  in het RNI (Register Niet Ingezetenen). In dat geval is het gezag niet te bepalen.
+
+  Gebruikte velden:
+    - Gemeente van inschrijving -> 08.09.10
+
+    Achtergrond:
+      Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Bert' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En 'Gerda' en 'Bert' zijn met elkaar gehuwd
+      En de persoon 'Zoe' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is in Nederland geboren
+      * heeft 'Gerda' als ouder 1
+      * heeft 'Bert' als ouder 2
+
+  Regel: Het kind is niet ingezetene in de BRP
+
+    Scenario: het kind staat niet als ingezetene in de BRP er is sprake van GezagNietTeBepalen
+      Gegeven persoon 'Zoe' 
+      * is niet ingeschreven in de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                               |
+      | type        | GezagNietTeBepalen                                                   |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet in Nederland woont. |
+
+  Regel: Het kind is wel ingezetene in de BRP
+
+    Scenario: het kind staat niet als ingezetene in de BRP er is sprake van TweehoofdigOuderlijkGezag
+      Gegeven persoon 'Zoe' 
+      * is ingeschreven in de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+  Regel: Het kan niet worden bepaald of het kind ingezetene in de BRP is
+
+    Scenario: Van het kind is niet bekend want zijn/haar verblijfplaats is er is sprake van GezagNietTeBepalen
```

Comment:
`zijn/haar verblijfplaats is...` moet zijn `zijn/haar verblijfplaats is onbekend`

Reply:
comments gecombineerd naar: Van een kind is de verblijfplaats onbekend is er is sprake van GezagNietTeBepalen

er is sprake van GezagNietTeBepalen bewaard voor de consistentie in de testen